infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/bootstrapper/KubernetesBootstrapper.java

```diff
@@ -161,14 +167,20 @@ private void injectBootstrapper() throws InfrastructureException {
     LOG.debug("Bootstrapping {}:{}. Downloading bootstrapper binary", runtimeIdentity, mName);
     exec(
         outputConsumer,
-        "curl",
-        // -f, --fail          Fail silently (no output at all) on HTTP errors
-        // -s, --silent        Silent mode
-        // -S, --show-error    Show error even when -s is used
-        // -o, --output <file> Write to file instead of stdout
-        "-fsSo",
-        BOOTSTRAPPER_DIR + BOOTSTRAPPER_FILE,
-        bootstrapperBinaryUrl);
+        "sh",
+        "-c",
+        "curl"
+            + (certProvisioner.isConfigured() ? " --cacert " + certProvisioner.getCertPath() : "")
+            +
+            // -f, --fail          Fail silently (no output at all) on HTTP errors
+            // -s, --silent        Silent mode
+            // -S, --show-error    Show error even when -s is used
```

Comment:
Do we need these comments?

Reply:
I think yes, because each time when I have some troubles with this `curl` command and see `-fsSo`,  I need to google each of option to check what may be wrong. So, I think it useful to see options description which we use.